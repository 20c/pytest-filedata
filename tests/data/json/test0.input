  {
    "_id": "5a4a580a0c912af0f37e0c90",
    "index": 0,
    "guid": "0962bf7e-3740-4e89-9b46-70ae0be4f191",
    "name": {
      "first": "Jane",
      "last": "Doe"
    },
    "range": [
      0,
      1,
      2,
      3,
      4,
      5,
      6,
      7,
      8,
      9
    ]
  }
