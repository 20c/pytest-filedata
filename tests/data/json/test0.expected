{"guid": "0962bf7e-3740-4e89-9b46-70ae0be4f191", "index": 0, "range": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9], "_id": "5a4a580a0c912af0f37e0c90", "name": {"last": "Doe", "first": "Jane"}}
